static/js/app.js

```diff
@@ -4,16 +4,17 @@ $(function () {
             "": "navigateEmpty",
             "/?marker=:id&start_date=:start&end_date=:end&show_fatal=:showFatal&show_severe=:showSevere&show_light=:showLight&show_inaccurate=:showInaccurate&zoom=:zoom&lat=:lat&lon=:lon": "navigate"
         },
-        navigate: function (id, start, end, showFatal, showSevere, showLight, showInaccurate, zoom, lat, lon) {
-            app.model.set("currentMarker", parseInt(id));
-            app.model.set("dateRange", [new Date(start), new Date(end)]);
-            app.model.set("showFatal", showFatal);
-            app.model.set("showSevere", showSevere);
-            app.model.set("showLight", showLight);
-            app.model.set("showInaccurateMarkers", showInaccurate);
-            app.map.setZoom(parseInt(zoom));
-            app.map.setCenter(new google.maps.LatLng(lat, lon));
-        },
+        //removed navigate function because we don't use backbone navigation for now
```

Comment:
I talked to @omerxx and @LironRS about this. The problem is that right now we have only some of the parameters updated in the URL so it doesn't fully represent the state of the map. Our idea was to get the "share current view" link and after loading the proper state of the map, remove all the parameters from the URL so we will have a "clean" one.
You are right about the history problem but maybe it can also be solved with saving the complete URL with all the parameters to the history.

What do you think?